examples/ingress-recommendations/README.md

```diff
@@ -0,0 +1,175 @@
+# Ingress recommendations
+
+This example scenario showcases the BotKube recommendations feature around Ingress resources.
+
+## Prerequisites
+
+### Software
+
+Install the following applications:
+
+- [colima](https://github.com/abiosoft/colima)
+- [Helm](https://helm.sh/)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+- [Mkcert](https://github.com/FiloSottile/mkcert)
+
+### Set up local Kubernetes cluster
+
+Run the command:
+
+```bash
+colima start --runtime containerd --kubernetes --cpu 2 --memory 3 --disk 20 --profile botkube
+```
+
+### Install Ingress Nginx Controller
+
+Run the command:
+
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace --wait \
+  -f ./examples/ingress-recommendations/ingress-overrides.yaml
+```
+
+### Generate and use self-signed certificate
+
+1. Generate the certificate:
+
+  ```bash
+  mkcert "*.botkube.local" botkube.local localhost 127.0.0.1 ::1
```

Comment:
It may not be good idea to trust mkcert file naming conventions, I would use following 
```
mkcert \
> -cert-file botkube.tls.crt \
> -key-file botkube.tls.key \
> "*.botkube.local" botkube.local localhost 127.0.0.1 ::1
```